core/templates/dev/head/pages/base.html

```diff
@@ -80,8 +80,6 @@
       var GLOBALS = {
         ADDITIONAL_ANGULAR_MODULES: JSON.parse(
           '{{additional_angular_modules|js_string}}'),
-        status_code: JSON.parse('{{status_code}}'),
-        GCS_RESOURCE_BUCKET_NAME: JSON.parse('{{GCS_RESOURCE_BUCKET_NAME|js_string}}'),
```

Comment:
I don't think this is an expect change.

Reply:
Yes, you are right. Mistakenly deleted this in a merge. Have added it back